### ISSUE TITLE
Repository for themes?

### ISSUE THREAD

**Issue by Baxtex** [Microsoft/VS-ColorThemes#4]
I have experimented a little bit but can't seem to make any of my changes work. I am looking for a dark theme with a little bit better contrast than the original dark theme.  Is there any repository for themes where you can download other themes?

**Comment by justcla** [comment]
There is no current repository for downloading other themes. This will become that place.
New themes will appear here over time.

I know that Mitch Muenster has been working on a Dark theme with Extra Contrast.
Hopefully he'll submit a PR soon!
Watch out for it. :-)

**Comment by MobileRez** [comment]
Yes I am working on getting a high contrast dark theme working I have not had the time to get my code integrated. Below are some screenshots with comparing the old dark theme (on right) and the new dark theme (on the left) that I am working on.

![darkcodecomparison](https://user-images.githubusercontent.com/6531190/31349962-e847a982-acea-11e7-9cfa-fa70a1634dc7.png)

![darkxamlcomparison](https://user-images.githubusercontent.com/6531190/31349961-e847785e-acea-11e7-8001-c73363713f25.png)

**Comment by Baxtex** [comment]
Looking good! I played around a bit more with the tool and managed to find the correct options to change. In my opinion, the text editor colors should be shown on  the "quickview" and not hidden. :)
I basically just made the existing colors a bit lighter, makes them alot easier to see when in an light enviroment! :D

**Comment by justcla** [comment]
Thanks Baxter.
Can you export your theme and upload it to a new branch so we can check out the editor colors you used?

**Comment by justcla** [comment]
Let's get a branch started, then we can add work together on making an awesome Dark High Contrast theme.

**Comment by Baxtex** [comment]
Yeah, sure. It's definatly not perfect, but it's a start. :D

**Comment by justcla** [comment]
I have created a new branch for the Dark theme with Extra Contrast:
https://github.com/Microsoft/VS-ColorThemes/tree/DarkExtraContrast

The initial draft is the DarkExtraContrast theme developed by Mitch Muenster.
Please download it and play with it.
If you can make some improvements to it, please submit PRs to the DarkExtraContrast branch.

Thanks for your contributions!

**Comment by MobileRez** [comment]
Those watching this thread, I have added my Theme and it got merged into the main branch [https://github.com/Microsoft/VS-ColorThemes/pull/9](https://github.com/Microsoft/VS-ColorThemes/pull/9)

**Comment by justcla** [comment]
@Baxtex , Now that Mitch's Dark theme with Extra Contrast has been uploaded and merged to master, did you want to improve it with your changes?
